src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -134,9 +138,16 @@ public void InitializeErrorUnit()
 
       public void SetDataFormat(IDataFormat format)
       {
-         _format = format;
-         _originalFormat = _format.Parameters.ToList();
-         setDataFormat(format.Parameters);
+         try
+         {
+            _format = format;
+            _originalFormat = _format.Parameters.ToList();
+            setDataFormat(format.Parameters);
+         }
+         catch
+         {
+            _dialogCreator.MessageBoxError(Error.ErrorSettingImportFormat);
```

Comment:
I am not in favor of catch all like this. Why is the exception not thrown from the setDataformat
Also we have an exceptionmanager that should be used to throw error. I was wondering why the error were not shown like other. It's because of the use of the dialogcreator directly

Reply:
> I am not in favor of catch all like this. Why is the exception not thrown from the setDataformat Also we have an exceptionmanager that should be used to throw error. I was wondering why the error were not shown like other. It's because of the use of the dialogcreator directly

@msevestre is definitely right. The error should have actually been thrown way sooner and as it happens, it already exists ;)